src/components/RichTextEditor/ReactEditorJS.tsx

```diff
@@ -0,0 +1,70 @@
+import EditorJS, {
+  EditorConfig,
+  OutputData,
+  ToolConstructable,
+} from "@editorjs/editorjs";
+import Paragraph from "@editorjs/paragraph";
+import {
+  EditorCore,
+  Props as ReactEditorJSProps,
+  ReactEditorJS as BaseReactEditorJS,
+} from "@react-editor-js/core";
+import React from "react";
+
+// Source of @react-editor-js
+class ClientEditorCore implements EditorCore {
+  private readonly _editorJS: EditorJS;
+
+  constructor({ tools, ...config }: EditorConfig) {
+    const extendTools = {
+      // default tools
+      paragraph: {
+        class: Paragraph,
+        inlineToolbar: true,
+      } as unknown as ToolConstructable,
+      ...tools,
+    };
+
+    this._editorJS = new EditorJS({
+      tools: extendTools,
+      ...config,
+    });
+  }
+
+  public async clear() {
+    await this._editorJS.clear();
+  }
+
+  public async save() {
+    return this._editorJS.save();
+  }
+
+  public async destroy() {
+    try {
+      await this._editorJS.destroy();
+    } catch (e) {
+      /*
+        Dismiss that error.
+        Sometimes instance is already unmounted while Editor wants to destroy it.
+        Editorjs does this properly so this error does not break anything
+       */
+    }
```

Comment:
**question**: is this only error that can be thrown here? If not, perhaps it would be good to specify which should be ignored

Reply:
seems like it's only this, if anything will rise up here, then we can let that through